terraform/cloudfoundry/cf_rds.tf

```diff
@@ -40,7 +40,7 @@ resource "aws_db_instance" "cf" {
   allocated_storage    = 10
   engine               = "postgres"
   engine_version       = "9.5.4"
-  instance_class       = "db.t2.small"
+  instance_class       = "db.m4.large"
```

Comment:
Any reason to use a m4.large over an m3.medium? The m3.medium is roughly half the cost of the m4.large.